admin/load.php

```diff
@@ -636,3 +639,33 @@ function perflab_plugin_admin_notices() {
 	}
 }
 add_action( 'admin_notices', 'perflab_plugin_admin_notices' );
+
+/**
+ * Callback function to handle admin custom script.
+ *
+ * @since n.e.x.t
+ */
+function perflab_enqueue_custom_script() {
+	?>
+<script>
+( function( $ ) {
+	if ( window.pagenow === 'settings_page_perflab-modules' ) {
+		// Only listen for ajaxComplete if this is the performance setting page.
+		$( document ).ajaxComplete( function( event, xhr, settings ) {
+			// Check if this is the 'install-plugin' request.
+			if ( settings.data && typeof settings.data === 'string' && settings.data.includes( 'action=install-plugin' ) ) {
+				setTimeout( function() {
+					var plugin_url = $( event.target.activeElement ).attr( 'href' );
+					var nonce = $( event.target.activeElement ).parents( '.plugin-card' ).attr( 'data-plugin' );
```

Comment:
As the attribute contains a nonce, `data-plugin` isn't a good name really. How about `data-activation-nonce` or `data-plugin-activation-nonce`?

Also see my other comment on trying to avoid the `parents()` lookup here.